common/credential-storage/src/storage.rs

```diff
@@ -30,9 +31,37 @@ pub trait Storage: Send + Sync {
     async fn consume_coconut_credential(
         &self,
         id: i64,
-        gateway_id: &str,
+    /// Inserts provided coin_indices_signatures into the database.
```

Comment:
Artefact of the PR stack, my bad.
We don't need the gateway_id anymore, because ticket book are by essence multi-use credential. You don't want to be able to spend the whole ticket book at a gateway